Exec/Make.PeleC

```diff
@@ -28,6 +28,12 @@ ifeq ($(USE_FORCING), TRUE)
   DEFINES += -DPELEC_USE_FORCING
 endif
 
+ifeq ($(USE_TURBINFLOW), TRUE)
```

Comment:
I'm not sure there really is a danger. But if we skip those we'll need to always have a `turb` object in the `probparm` or find another mechanism to check if we want to inject turbulence or not.

Reply:
Ok yeah I see the problem... Do you think it would make sense to orchestrate the turbulence parm stuff outside of the prob stuff? It feels like if we have a bunch of these problems then they have a whole bunch of duplicate code.

Also I think we only need a host version of the turbulence parm stuff. So if we lift it out of the probparm then we can avoid all the GPU-CPU copies. Or am I missing something?

Reply:
we can use `tp.turbinflow_initialized` to detect if the code should call the turbulence inflow. so we don't need `do_turb`